src/views/Home/index.view.tsx

```diff
@@ -1,9 +1,16 @@
 import React from "react";
+import HeroView from "views/Hero/index.view";
+import Navbar from "components/Navbar/index.view";
 
 import "./Home.scss";
 
 const HomepageView: React.FC = () => {
-  return <div className="Homepage">Hello from HomepageView</div>;
+  return (
+    <div className="Homepage">
+      <Navbar />
```

Comment:
If each `Hero` view has its own theme (Home/Landing, Schedule, etc.), is the `Navbar` component going to be themed as well or static across views. 

If `Schedule` has a orange/pink gradient, we might want to make the navigation bar consistent with that - not too sure how the design will incorporate that though!

Reply:
Yeah so if you take a look at [the design](https://www.figma.com/file/9vYrREdbNaxL3bLlyHnR7m/CruzHacks-2021-Website?node-id=130%3A1) thus far it looks like the Navbar is just plain white. I don't imagine it changing. Unfortunately, there is a lot in the air right now since we do not have the high fidelity mockup for the entire UI. If the Navbar changes with each view, we will follow the same pattern to add a `pageName` prop to that Navbar component and have the navbar component do the logic to change colors or do whatever it needs to do.

Reply:
Well, I was talking mainly about the font color - right now, it looks blue as if it matches the background of postcard gradient. If other views have different gradients, will the font color be subject to change?

Reply:
Not entirely sure if I follow..this story is a simple hotfix to create a hero component so you guys can work on implementing your component. I will work on navbar in a separate feature story. I just imported what needed to be imported in the specific files and then you guys style and create the components.

Reply:
Understandable - I was looking into the persistence of the implementation, but we can worry about that later.